Projects/App/Sources/AppDelegate+Firebase.swift

```diff
@@ -51,7 +50,7 @@ extension AppDelegate: MessagingDelegate {
 extension AppDelegate {
     func configureDebuggingFB(application: UIApplication) {
         guard let filePath = Bundle.main.path(
-            forResource: "GoogleService-debugging-Info",
+            forResource: "GoogleService-Info-debugging",
             ofType: "plist"
         ),
               let options = FirebaseOptions(contentsOfFile: filePath)
```

Comment:
```suggestion
    func configureFirebase(application: UIApplication) {
        var googleInfoName: String
        #if DEBUG
        googleInfoName = "GoogleService-Info"
        #else
        googleInfoName = "GoogleService-Info-debugging"
        #endif
        guard let filePath = Bundle.main.path(
            forResource: googleInfoName,
            ofType: "plist"
        ),
              let options = FirebaseOptions(contentsOfFile: filePath)
        else { return }
        FirebaseApp.configure(options: options)
        application.registerForRemoteNotifications()
    }
```
기존의 함수에서 전처리문으로 리소스 경로만 수정하는게 어떨까요?

Reply:
이 버전 코드가 앱 델리게이트쪽 코드를 더 깔끔하게 해줄 것 같네요 수정하도록 하겠습니다 ~